src/tests/functional/plugin/conformance/subgraphs_dumper/src/op_cloner.cpp

```diff
@@ -72,13 +72,13 @@ void get_port_range(const std::shared_ptr<ov::op::v0::Constant> &constant_input,
 
 std::shared_ptr<ov::Node> clone(const std::shared_ptr<ov::Node> &node, LayerTestsUtils::OPInfo &meta) {
     ov::OutputVector op_inputs;
-    bool static_inputs = true;
+    bool has_parameters = false;
     for (size_t i = 0; i < node->get_input_size(); ++i) {
         const auto input = node->input(i).get_source_output();
-        static_inputs &= input.get_partial_shape().is_static();
         auto port_info = LayerTestsUtils::PortInfo();
         const auto constant = ov::get_constant_from_source(input);
         if (constant != nullptr) {
+            has_parameters = true;
```

Comment:
Is it right that has_parameters = true; here, but not in else ?

Reply:
Fixed